docs/buildrun.md

```diff
@@ -49,6 +49,7 @@ The `BuildRun` definition supports the following fields:
 - Optional:
   - `spec.resources` - Refers to the compute [resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) used on the container where the image is built.
   - `spec.serviceAccount` - Refers to the SA to use when building the image. (_defaults to the `default` SA_)
+  - `spec.timeout` - Defines a custom timeout. The value needs to be parsable by [ParseDuration](https://golang.org/pkg/time/#ParseDuration), for example `5m`. The default overwrites the value that is defined in the `Build`.
```

Comment:
Shouldn't this be - `Can be overwritten in the BuildRun` instead of `Can be overwritten in the Build.` considering the file is `docs/buildrun.md`

Reply:
No, but still is a typo. Should be `The value overwrites the value that is defined in the `Build`. I'll open another PR with an improved wording. I'll make you reviewer.

Reply:
@DhritiShikhar let's discuss there: https://github.com/redhat-developer/build/pull/201